### ISSUE TITLE
Unable to select/focus on the NHS logo in the header using VoiceOver on iOS

### ISSUE THREAD

**Issue by mcheung-nhs** [nhsuk/nhsuk-frontend#631]
# Bug Report

## What is the issue?
When using VoiceOver on iOS, the NHS logo in the header is not selectable.

## What steps are required to reproduce the issue?
On an iPhone or iPad, Turn VoiceOver on, go to nhs.uk on Safari, then try to swipe to the NHS logo or tap on the NHS logo

## What was the environment where this issue occurred?

* Device: iPhone 8
* Operating System: iOS 13.5.1
* Browser: Safari
* Browser version: 13.5.1

* Device: iPad (6th gen)
* Operating System: iOS 12.2
* Browser: Safari
* Browser version: 12.2

**Comment by davidhunter08** [comment]
@mcheung-nhs could this be to do with Safari not enabling tab highlighting by default? 

See https://www.seanmcp.com/articles/tab-focus-not-working-in-safari/

**Comment by mcheung-nhs** [comment]
@davidhunter08 - I think that is more Safari on desktop, rather than on iOS.

**Comment by mcheung-nhs** [comment]
The cause of this is the IE png fallback along with some attributes in the `svg`. This will be fixed as part of https://github.com/nhsuk/nhsuk-frontend/pull/680 so awaiting for that to be released.

**Comment by AdamChrimes** [comment]
This will be fixed in release 5.0.0 (next week)
